### DISCUSSION TITLE
SK BBAN check

### DISCUSSION THREAD

**Discussion by jaroborovsky** [Simplify/ibantools#234] [Ideas]
Hi,
can you improve Country specification for SK - Slovakia?
Slovakia has same BBAN validation algorithm as Czech republic.  
[nbs.sk document in Slovak](https://nbs.sk/_img/documents/_legislativa/_vestnik/opat8-09.pdf)
SK: { ... bban_validation_func: checkCzechBBAN, ..}
Thanks.

**Comment by Simplify** [comment]
#238 - implemented and merged. Released as 4.2.0
